docs/content/docs/start/tutorial.md

```diff
@@ -96,78 +100,89 @@ async function start() {
+### Starting with a static view
 
-It is simplest to create the view using local data without Fluid, then add Fluid by changing some key pieces of the app. This tutorial uses this approach.
-
-The `renderDiceRoller` function appends the `diceTemplate` to the passed in HTML element, and creates a working dice roller with a random dice value each time the "Roll" button is clicked. The `diceMap` will be used in the next few steps.
+It is simplest to create the view using local data without Fluid, then add Fluid by changing some key pieces of the app. We used this approach to create the dice roller app. The following is the Fluid-less view. Note that the `renderDiceRoller` function appends the `diceTemplate` to the passed in HTML element, and creates a working dice roller with a random dice value, from 1 to 6, each time the "Roll" button is clicked.
 
 ```js
 const diceTemplate = document.createElement("template");
 
 diceTemplate.innerHTML = `
+  <style>
+    .wrapper { text-align: center }
+    .dice { font-size: 200px }
+    .roll { font-size: 50px;}
+  </style>
   <div class="wrapper">
     <div class="dice"></div>
     <button class="roll"> Roll </button>
   </div>
 `
-function renderDiceRoller(diceMap, elem) {
+function renderDiceRoller(elem) {
     elem.appendChild(diceTemplate.content.cloneNode(true));
     const rollButton = elem.querySelector(".roll");
     const dice = elem.querySelector(".dice");
 
-    rollButton.onclick = () => updateDice(Math.floor(Math.random() * 6)+1);
+    rollButton.onclick = () => updateDice(Math.floor(Math.random() * 6)+1);
 
-    const updateDice = (value) => {
-        // Unicode 0x2680-0x2685 are the sides of a die (⚀⚁⚂⚃⚄⚅).
-        dice.textContent = String.fromCodePoint(0x267f + value);
-    };
+    const updateDice = (value) => {
+        // Unicode 0x2680-0x2685 are the sides of a die (⚀⚁⚂⚃⚄⚅).
+        dice.textContent = String.fromCodePoint(0x267f + value);
+        dice.style.color = `hsl(${diceValue * 60}, 70%, 30%)`;
+    };
     updateDice(1);
 }
 ```
 
-## Connect the view to Fluid data
+## Connecting the view to Fluid data
 
 ### Modifying Fluid data
 
-To begin using Fluid in the application, the first thing to change is what happens when the user clicks the `rollButton`. Instead of updating the local state directly, the button updates the number stored in the `value` key of the passed in `diceMap`. Because the `diceMap` is a Fluid `SharedMap`, changes will be distributed to all clients. Any changes to the `diceMap` will cause a `valueChanged` event to be emitted, and an event handler can trigger an update of the view.
+To begin using Fluid in the application, the first thing to change is what happens when the user clicks the `rollButton`. Instead of updating the local state directly, the button updates the number stored in the `value` key of the Fluid `diceMap` object that we created earlier. So, we added the `diceMap` as an additional parameter to the `rollButton` method.
```

Comment:
The other step-by-step guides seem to be written in past tense, but this one is in present tense.

Reply:
Could you elaborate? I'm not sure what you mean. Would it help if the 2nd sentence said "Instead of updating the local state directly, the button **should update** the number stored in the `value` key of the Fluid `diceMap` object that we created earlier. "


Also, this isn't a step-by-step in the usual sense of having the reader take actions. (Step 1: do this. Step 2: do that.) It's really just a description of the app.js file, section-by-section. I'm not clear about how we would do that in the past tense.

Reply:
Oh, sorry. I just mean in the other steps we say things like "Next, we needed to revise the `rollButton` method". I'm just wondering if this should be something like "To begin using Fluid in the application, the first thing we needed to change was...".

Reply:
I see. I'll fix that.